lib/wice/helpers/wice_grid_view_helpers.rb

```diff
@@ -154,9 +154,7 @@ def generate_blank_slate(grid, rendering) #:nodoc:
         rendering.blank_slate_handler
       end
 
-      if rendering.find_one_for(:in_html) { |column| column.detach_with_id }
-        grid.output_buffer.return_empty_strings_for_nonexistent_filters = true
-      end
+      rendering.find_one_for(:in_html, true) { |column| column.detach_with_id }
```

Comment:
this will cause an error. Wice::GridRenderer#find_one_for has the arity of 1.
I think you want to pass `true` to the output_buffer, but rendering is a different object